dll/kernel32.cpp

```diff
@@ -1823,6 +1828,19 @@ namespace kernel32 {
 		}
 	}
 
+	int WIN_FUNC GetLocaleInfoW(unsigned int Locale, int LCType, wchar_t *lpLCData, int cchData) {
+		DEBUG_LOG("GetLocaleInfoW %d %d\n", Locale, LCType);
+		std::string info = str_for_LCType(LCType);
+		std::wstring ret(ret.begin(), ret.end());
```

Comment:
info.begin(), info.end()? Would be better to use one of the string -> wstring conversion functions we have though, rather than hard-code the naive encoding we use in one more place